### ISSUE TITLE
server.log mob moved wrongly!! and mounts get teleported with player

### ISSUE THREAD

**Issue by talhanation** [Alex-the-666/Ice_and_Fire#1904]
Hello, 
this is a major bug in multiplayer wich is killing the gameplay...

This happens everytime when a player wich rides somthing gets close to a nother player (around 50 blocks) 
the server will spam: ( ... moved wrongly) and the player who rides will get teleported around and get mostly killed thru fall damage,
 
i have tested it on all mounts and also vanilla horses wich get hit by this bug too. 

I hope you will fix this soon :D, thank you for this incredible mod guys and have a nice day!

Client and Sever:
Mod Version: 1.8.2
Forge Version:14.23.5.2846

i have tested this only with Ice and Fire mod installed.
if needed i can make a video and reproduce the error/bug

**Comment by ghost** [comment]
The same problem here, and the same porblem described by `Nitesurgeon ` on [curseforge](https://www.curseforge.com/minecraft/mc-mods/gotta-go-fast) (the last comment):


> I am experiencing an issue with my custom modpack forge server, where when I travel on some kind of mount (horses, Hippogryphs, boats, Chocobos, etc.) I will start violently rubber-banding back and forth while the server console spams "___ moved wrongly!". I have done alot of digging around and what I understand is this has to do with the server side checks that prevent players from moving too fast. However, none of the solutions online seem to work for me. I have made sure to allow flight to true, I have disabled the ElytraMovementCheck gamerule, and have tried both the MTQFix and Gotta Go Fast mods.
> Something I have noticed is that I only have this problem when attempting to ride/fly on these mounts around other players on the server. If I am alone,or far away from other players I can do it no problem. So my guess is the problem lies around how the server handles movement speed around multiple players. I had found a reddit post a few days back that describes this issue of rubber banding around other players, but have since lost it. Anyone have any suggestions as to what might be happening? I will post my modpack, general specs, and my server batch file text below:
> My Modpack: https://www.curseforge.com/minecraft/modpacks/tales-of-the-engineer-herbalist-and-mage
> Modlist: https://pastebin.com/eHLXwaMU

**Comment by Akhorishaan** [comment]
This issue is still happening on 1.8.3

**Comment by hakasapl** [comment]
Confirm this is happening with hippogryphs and amphitheatre, the dragons seem to be fine. I did some investigating on my own about this, hopefully it helps your troubleshooting.

I tried running Ice and Fire on a sponge server (since sponge has an option to disable all "moved wrongly" or "moved too quickly" actions), but I got an error only with the movement module enabled. I believed it was conflicting with the core mod that came with ice and fire, so I modified version 1.8.3 to not include the core mod. I tried again, and all seemed to be fine, however, tamed hippogryphs now started dissapearing in the air while flying, never to be seen again. (Not sure if that gives you any hints, but I thought I'd let you know.)
